tests/comparisons/Migration/pgsql/test_snapshot_plugin_blog_pgsql.php

```diff
@@ -117,6 +168,143 @@ public function up(): void
             ])
             ->create();
 
+        $this->table('products')
+            ->addColumn('title', 'string', [
+                'default' => null,
+                'limit' => 255,
+                'null' => true,
+            ])
+            ->addColumn('slug', 'string', [
+                'default' => null,
+                'limit' => 100,
+                'null' => true,
+            ])
+            ->addColumn('category_id', 'integer', [
+                'default' => null,
+                'limit' => 10,
+                'null' => true,
+            ])
+            ->addColumn('created', 'timestamp', [
+                'default' => null,
+                'limit' => null,
+                'null' => true,
+                'precision' => 6,
+                'scale' => 6,
+            ])
+            ->addColumn('modified', 'timestamp', [
+                'default' => null,
+                'limit' => null,
+                'null' => true,
+                'precision' => 6,
+                'scale' => 6,
+            ])
+            ->addIndex(
+                $this->index([
+                        'id',
+                        'category_id',
+                    ])
+                    ->setName('products_category_unique')
+                    ->setType('unique')
+            )
+            ->addIndex(
+                $this->index('slug')
+                    ->setName('products_slug_unique')
+                    ->setType('unique')
+            )
+            ->addIndex(
+                $this->index('title')
+                    ->setName('products_title_idx')
+            )
+            ->create();
+
+        $this->table('special_pks', ['id' => false, 'primary_key' => ['id']])
```

Comment:
Is this the expected behavior now? Generating a migration for a plugin without `--require-table` will include _every_ table in the database?

Reply:
Check the initial report. I think it does make sense as outlined above, since:
- One can use --require-table if you want to filter by existing Table classes